core/arch/arm/plat-zynqmp/conf.mk

```diff
@@ -23,6 +23,9 @@ endif
 # ZynqMP supports up to 40 bits of physical addresses
 CFG_CORE_ARM64_PA_BITS ?= 40
 
+# Configure DDR size either by using device tree or override it with:
+# CFG_DDR_SIZE ?= 0x80000000
```

Comment:
A new `CFG_` should be documented somewhere, for instance `mk/config.mk`
`CFG_DDR_SIZE` sounds a bit ambiguous. Is it the total amount of physical DDR? Including eventual secure and possibly other reserved portions? Or is it only the non-secure parts usable by normal world?

Reply:
CFG_DDR_SIZE is used in:
- plat-imx/conf.mk:CFG_DDR_SIZE
- plat-stm/conf.mk:CFG_DDR_SIZE

And in those define it as being bytes of available DDR memory. So I have continued the treat.

Other similar things that I spotted:
- plat-hikey/conf.mk:CFG_DRAM_SIZE_GB
- plat-poplar/conf.mk:CFG_DRAM_SIZE_GB
- plat-ls/conf.mk:CFG_DRAM0_SIZE
- plat-uniphier/conf.mk:CFG_DRAM0_SIZE
- plat-mediatek/conf.mk:CFG_DRAM_SIZE
- plat-stm32mp1/conf.mk:CFG_DRAM_SIZE

Probably would be a good idea to commonize these a bit and document them.

I selected the DDR variant as that's how it is called in manuals too.

In file `./core/arch/arm/include/mm/generic_ram_layout.h` there are defines that are used to configure secure RAM locations and sizes so I would re-use those for defining other features.

As there is a need to configure DDR dimensions for OP-TEE then I suppose device tree and if that is not available then configurations like DDR memory size should be used.

Reply:
If you have direction you want to go I can add necessary notes to `mk/config.mk`. Thou probably not a good idea to set default value for it.

Reply:
i think your current implementation (using `CFG_DDR_SIZE` straight from your _plat-xxx/conf.mk_) is fine.
What's done with these `CFG_DRAM_SIZE`'-like config switches is really platform specific. Unless there is generic code using a unified `CFG_DDR_{BASE|SIZE}` or like, i don't think there is value factorizing the platforms configs (IMHO).

Reply:
Fair enough.